src/components/ebay-table/index.marko

```diff
@@ -73,9 +74,7 @@ $ const {
                             sortEleAttr = { href };
                         } else if (sortOrder) {
                             sortEleAttr = {
-                                type: "button",
```

Comment:
I belive we still need to add an a11y text for screen readers to know what the sort order is. 
Lets chat tomorrow